challenge.md

```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git clone url
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git status
 
 <!-- Answer Ends Here -->
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git commit --amend
 
 <!-- Answer Ends Here -->
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->origin...not sure.
```

Comment:
`origin` refers to our fork - the repo that our fork was copied from is usually referred to as `upstream`. And to grab changes from someplace else and incorporate them into our current project, the command is `git pull`. 

So, tying it all together, you'd need to type `git pull upstream` (plus, optionally, a branch, such as `master`).